### PR TITLE
docs: clarify about error in shada

### PR DESCRIPTION
**Pull request by sisrfeng**
Small fix.

### CODE REVIEW

File: runtime/doc/starting.txt [hunk 1]
```diff
@@ -1337,10 +1338,13 @@ exactly four MessagePack objects:
...
-Errors in ShaDa file may have two types: E575 used for all “logical” errors
-and E576 used for all “critical” errors.  Critical errors trigger behaviour
-described in |shada-error-handling| when writing and skipping the rest of the
-file when reading and include:
+Errors in ShaDa file may have two types: 
+1. E575 for “logical” errors.
+2. E576 for “critical” errors.  
+When writing, critical errors trigger behaviour described in 
+|shada-error-handling|.
+When reading, critical errors trigger behaviour skipping the rest of the file.
```
Comment (zeertzjq) [inline runtime/doc/starting.txt:1346]:
```suggestion
When reading, critical errors cause the rest of the file to be skipped.
```

File: runtime/doc/starting.txt [hunk 2]
```diff
@@ -1119,7 +1119,8 @@ any free letter from `a` to `z`) while normally it will create this file,
 write to it and then rename `{basename}.tmp.X` to `{basename}`. Such errors
 include:
 
-- Errors which make Nvim think that read file is not a ShaDa file at all:
+- Errors which make Nvim think that the file being read is not a ShaDa
```
Comment (zeertzjq) [inline runtime/doc/starting.txt:1122]:
```suggestion
- Errors which make Nvim think that the file being read is not a ShaDa
```

File: runtime/doc/starting.txt [hunk 3]
```diff
@@ -1337,10 +1338,13 @@ exactly four MessagePack objects:
...
 
 								*E575* *E576*
-Errors in ShaDa file may have two types: E575 used for all “logical” errors
-and E576 used for all “critical” errors.  Critical errors trigger behaviour
-described in |shada-error-handling| when writing and skipping the rest of the
-file when reading and include:
+Errors in ShaDa file may have two types: 
+1. E575 for “logical” errors.
+2. E576 for “critical” errors.  
+When writing, critical errors trigger behaviour described in
```
Comment (zeertzjq) [inline runtime/doc/starting.txt:1344]:
```suggestion
When writing, critical errors trigger behaviour described in
```

File: runtime/doc/starting.txt [hunk 4]
```diff
@@ -1337,10 +1338,13 @@ exactly four MessagePack objects:
                        reasons, see |shada-compatibility|.
 
 								*E575* *E576*
-Errors in ShaDa file may have two types: E575 used for all “logical” errors
-and E576 used for all “critical” errors.  Critical errors trigger behaviour
-described in |shada-error-handling| when writing and skipping the rest of the
-file when reading and include:
+Errors in ShaDa file may have two types:
```
[the remark this answers is not in the document]
Comment (sisrfeng) [reply runtime/doc/starting.txt:1341]: Could you help me commit your changes directly? I'm not so familiar with PR.

**Pull request merged**